app/assets/stylesheets/pageflow/mixins/buttons.css.scss

```diff
@@ -25,4 +25,73 @@
     font-size: 16px;
     vertical-align: middle;
   }
-}
\ No newline at end of file
+}
+
+@mixin simple-button($base-color) {
+  $grayscale:     $base-color == grayscale($base-color);
+  $color:         hsl(0, 0, 100%);
+  $border:        adjust-color($base-color, $saturation:  9%,  $lightness: -14%);
+  $inset-shadow:  adjust-color($base-color, $saturation: -8%,  $lightness:  15%);
```

Comment:
We should put the space in front of the number to align the commas (and prevent commas followed by multiple spaces), here and in the line above.